app/views/common/_discourse_stylesheet.html.erb

```diff
@@ -3,7 +3,8 @@
 <%- if rtl? %>
   <%= discourse_stylesheet_link_tag(mobile_view? ? :mobile_rtl : :desktop_rtl) %>
 <%- else %>
-  <%= discourse_stylesheet_link_tag(mobile_view? ? :mobile : :desktop) %>
+  <%= discourse_stylesheet_link_tag(:mobile, { media: "all and (max-width: 450px)" }) %>
+  <%= discourse_stylesheet_link_tag(:desktop, { media: "all and (min-width: 450px)" }) %>
```

Comment:
Nitpick, one of these two rules should be set to `!450px`, because at exactly 450px, both stylesheets would be loaded, right?

Reply:
Can we use a [range syntax](https://caniuse.com/css-media-range-syntax) here?